cugl/lib/net/CUNetworkConnection.cpp

```diff
@@ -72,10 +72,11 @@ CUNetworkConnection::CUNetworkConnection(const ConnectionConfig& config)
 	remotePeer = HostPeers(config.maxNumPlayers);
 }
 
-CUNetworkConnection::CUNetworkConnection(const ConnectionConfig& config, std::string roomID)
+CUNetworkConnection::CUNetworkConnection(const ConnectionConfig& config, std::string newRoomId)
 	: status(NetStatus::Pending), apiVer(config.apiVersion), numPlayers(1), maxPlayers(0) {
 	c0StartupConn(config);
-	remotePeer = ClientPeer(std::move(roomID));
+	roomID = newRoomId;
```

Comment:
could you put this in our game's logic. Because if the library gets updated this code will get overwritten

Reply:
how would I do this since `getRoomId` in `NetworkMessageManager` calls `_conn->getRoomId`?